Cargo.toml

```diff
@@ -77,4 +77,4 @@ required-features = ["random"]
 [[test]]
 name = "test-predicate"
 path = "tests/predicate.rs"
-required-features = ["random"]
+required-features = ["random"]
```

Comment:
```suggestion
required-features = ["random"]

```

Reply:
oh, done cargo sort down bellow that fixed this.